react/week1/todolist/src/TodoItems.js

```diff
@@ -0,0 +1,17 @@
+import { todos } from "./TodoList";
+
+export default function Items() {
+    return (
+        <main>
+            <ul style={{listStyle:"none"}}>
+                {todos.map(todo => <TodoListItem todo={todo} key={todo.id}/>)}
+            </ul>
+        </main>
+    )
+}
+
+function TodoListItem ({todo}) {
+    return (
+        <li>* {todo.description}, {todo.deadline}</li>
+    )
+}
```

Comment:
A better naming can be used here, `Items` is a bit too general, what about `TodoList` ? what do you think? since the list item component is called `TodoListItem`, it can be a perfect match